piccolo/table.py

```diff
@@ -981,6 +981,17 @@ def create_table(
             only_default_columns=only_default_columns,
         )
 
+    @classmethod
+    def drop_table(cls):
+        """
+        Drop table, along with all columns.
+
+        .. code-block:: python
+
+            await Band.drop_table()
+        """
+        return drop_db_tables(cls)
```

Comment:
Instead of this, it could be a proxy to `alter().drop_column()` - that way it can be used sync or async.

```python
    @classmethod
    def drop_table(cls):
        """
        Drop table, along with all columns.
        .. code-block:: python
            await Band.drop_table()
        """
        return cls.alter().drop_table()
```

So user's can then do either of these:

```python
# async
await MyTable.drop_table()

# sync
MyTable.drop_table().run_sync()
```

The main question though is whether to add this. It's a trade-off. The pro is it's more convenient. The con, is it's more likely someone will accidentally call it by accident. Hmm, tricky 🤔

Reply:
Since the main issue was that I thought the feature was missing, I think things could be improved by making improvements to the documentation. Perhaps having a quick start page that has a list of common how tos. Something like this would work:

### I Want To:

* Define a new table: [code with explanation]
* Create a table: [code with explanation]
* Drop a table: [code with explanation]
* Perform a query: [code with explanation]
* Perform a join on a query: [code with explanation]

It's not that the documentation isn't there, it's just that there didn't seem to be a quick and easy cheat sheet which I think would really benefit new users (such as me).